res/css/views/rooms/_EventTile.scss

```diff
@@ -905,6 +906,12 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
             }
         }
 
+        .mx_MLocationBody_error {
```

Comment:
This error should be displayed the same as the generic error `mx_EventTile_tileError` in thread view.

Reply:
It is to avoid this:

![temp](https://user-images.githubusercontent.com/3362943/166207540-e8687605-7dea-4c3b-a130-d8a557dc5f29.png)

Reply:
`span` could be replaced with `div` for it but it is not sure what it would cause.